src/bootstrapper/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+set(BOOTSTRAPPER_TARGET_NAME bootstrapper)
+
+configure_file(bootstrapper.rc.in resources/bootstrapper.rc)
+
+file(GLOB BOOTSTRAPPER_HEADERS LIST_DIRECTORIES=false *.h)
+file(GLOB BOOTSTRAPPER_SOURCES LIST_DIRECTORIES=false *.cpp)
+file(GLOB BOOTSTRAPPER_RESOURCES LIST_DIRECTORIES=false ${CMAKE_CURRENT_BINARY_DIR}/resources/*.rc)
+
+add_executable(${BOOTSTRAPPER_TARGET_NAME}
+	WIN32
+	${BOOTSTRAPPER_HEADERS}
+	${BOOTSTRAPPER_SOURCES}
+	${BOOTSTRAPPER_RESOURCES}
+)
+
+target_compile_features(${BOOTSTRAPPER_TARGET_NAME}
+	PRIVATE
+		cxx_std_17
+)
+
+target_compile_definitions(${BOOTSTRAPPER_TARGET_NAME}
+    PRIVATE
+        UNICODE
+)
+
+set_target_properties(${BOOTSTRAPPER_TARGET_NAME}
+	PROPERTIES
+		RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/${BOOTSTRAPPER_TARGET_NAME}/${CMAKE_BUILD_TYPE}"
+		MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>"
+)
+
+install(TARGETS ${BOOTSTRAPPER_TARGET_NAME}
+	RUNTIME DESTINATION bin

```

Comment:
AFAIK, `bin` directory is the default `RUNTIME` destination (and, anyway, this is configured by `cmake`/`CMakeToolchain`), so I think we don't need this line.